src/FrontDoor/FrontDoor/Cmdlets/SetAzureRmFrontDoorFireWallPolicy.cs

```diff
@@ -87,7 +84,23 @@ public class SetAzureRmFrontDoorFireWallPolicy : AzureFrontDoorCmdletBase
         [Parameter(Mandatory = false, HelpMessage = "Managed rules inside the policy")]
         public PSManagedRule[] ManagedRule { get; set; }
 
+        /// <summary>
+        /// Redirect URL used for redirect actions
+        /// </summary>
+        [Parameter(Mandatory = false, HelpMessage = "Redirect URL")]
+        public string RedirectUrl { get; set; }
 
+        /// <summary>
+        /// Custom block response code used for block actions
+        /// </summary>
+        [Parameter(Mandatory = false, HelpMessage = "Custom Response Status Code")]
+        public ushort? CustomBlockResponseStatusCode { get; set; }
```

Comment:
Same comment as above - this should be an int with a ValidateRange attribute

Reply:
Fixed in next iteration.

Reply:
Again, this is a breaking change in the next iteration.  Please fix.

Reply:
By iteration, I meant in this code review... I already fixed this (made an int like you suggested).

Reply:
Also, just to be clear about something... I would have rather leave the ushort? because that allowed to set it back to $null while doing an integer with a range does not allow that. It's not a big deal since the customer could always type the default status code (but it would have been nice to have a nullable int with a range). So, there is a valid reason to sometimes use nullable parameters in arguments (when you want the user to be able to unset an optional property).

Reply:
Reopening this conversation since I think this should really be an optional null-able int anyway. I don't still understand why you are against null-able parameters when it seems to be the only way to unset an optional property.

Reply:
If you are not replacing all values with the values the user provides, then the verb for this cmdlet shoudl be Update and not Set.  Set indicates that the values you are providing are replacing the value on the service.  For Patch-like semantics, you should use Update.

If you are replacing, then absence of the bound parameter indicates that the value is cleared.